examples/webgl_loader_ldraw.html

```diff
@@ -244,6 +250,183 @@
 
 			}
 
+			function mergeObject( object ) {
```

Comment:
What do you guys think about moving this code into a static method on `LDrawLoader` and keep the example code simple?

Reply:
Yes, I was thinking the same. The loader code has been simplified by extracting this functionality to the function. I will do it in some hours.

Reply:
Done!